app/controllers/support_interface/performance_controller.rb

```diff
@@ -1,5 +1,21 @@
+require 'csv'
+
 module SupportInterface
   class PerformanceController < SupportInterfaceController
     def index; end
+
+    def application_timings
+      applications = SupportInterface::ApplicationsExport.new.applications
+
+      csv = CSV.generate do |rows|
+        rows << applications.first.keys
+
+        applications.each do |a|
+          rows << a.values
+        end
+      end
+
+      render plain: csv
```

Comment:
If we set `Content-Type` clients ought to download it rather than rendering it?

Reply:
I find it easier to copy-paste into Google Sheets than to import it, so I'd like to keep this for now until Matteo says otherwise.